code/modules/chemistry/Chemistry-Dispenser.dm

```diff
@@ -523,11 +523,13 @@ TYPEINFO(/obj/machinery/chem_dispenser)
 	dispense_sound = 'sound/misc/pourdrink2.ogg'
 
 /obj/machinery/chem_dispenser/chef
-	name = "kitchen fountain"
-	desc = "A soda fountain that definitely does not have a suspicious similarity to the alcohol and chemical dispensers OR the soda fountain. No sir."
-	dispensable_reagents = list("cola", "juice_lime", "juice_lemon", "juice_orange", "mint", "mustard", "pepper", \
-								"juice_cran", "juice_cherry", "juice_pineapple","coconut_milk", "ketchup", \
-								"sugar", "water", "vanilla", "tea", "chocolate", "chocolate_milk","strawberry_milk")
+	name = "HAPPY CHEF Dispense-o-tronic"
+	desc = "It's covered in a thin layer of acrid-smelling dust. The contents probably taste more like preservatives than whatever they're supposed to be."
+	dispensable_reagents = list("juice_cherry", "chocolate", "chocolate_milk", "coconut_milk", \
+								"cola", "juice_cran", "ginger_ale", "gravy", "grenadine", "ketchup", \
+								"juice_lemon", "juice_lime", "mustard", "juice_orange", \
+								"pepper", "juice_pineapple", "strawberry_milk", "sugar", \
+								"tea", "juice_tomato", "vanilla")
```

Comment:
```suggestion
	dispensable_reagents = list("ketchup","mustard","salt","pepper","gravy","chocolate","mint","chocolate_milk","strawberry_milk")
```

Reply:
Standardizing to the Nadir list readds mint and salt which I was specifically asked to remove earlier. If that decision's changed, that's great, just letting you know